### DISCUSSION TITLE
Howto add an endpoint description ?

### DISCUSSION THREAD

**Discussion by wiquanAppD** [usebruno/bruno#524] [Q&A]
Is there a way to add a endpoint description to the metadata that would be rendered in the IDE ?

eg.
"description": "Returns the version, loaded extensions, memory used, total memory and other useful information about the node."

**Comment by joswr1ght** [comment]
I also wish Bruno did a better job populating documentation, examples, descriptions, and other useful elements when importing OpenAPI files. To help with this, I wrote a Python script to automatically populate the `.bru` files `docs { }` element with OpenAPI endpoint details.

Usage:

1. Open OpenAPI v3 file with Bruno.
2. Exit Bruno.
3. Run `autobrunodoc doc -i openapi.yaml -w /path/to/bruno/workspace` (this will extract details from the OpenAPI file and populate the `.bru` documentation for each endpoint path.
4. Start Bruno.
5. Navigate to endpoint Docs tab to see documentation.

Available here: https://github.com/joswr1ght/autobrunodoc, feedback welcome.

![image](https://github.com/user-attachments/assets/16079e85-044c-415e-b823-cfb4dc21a780)
